drivers/ethernet/eth_xilinx_axienet.c

```diff
@@ -0,0 +1,577 @@
+
+#define LOG_MODULE_NAME eth_xilinx_axienet
+#define LOG_LEVEL       CONFIG_ETHERNET_LOG_LEVEL
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(LOG_MODULE_NAME);
```

Comment:
nit but please do the log module register as one line (it can take two arguments for the name and level, dont need to define macros)

Reply:
Done